code/src/index.css

```diff
@@ -11,3 +14,124 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+h1 {
+  text-align: center;
+  padding-bottom: 16px;
+  border-bottom: 1px solid rgb(60, 60, 60);
+}
+h2 {
+  font-size: 18px;
+}
+.container {
+  display: grid;
+  justify-content: center;
+  margin: 0 auto;
+  width: 90%;
+}
+.album {
+  max-width: 300px;
+  padding: 16px;
+  background-color: black;
+}
+img {
+  width: 100%;
+  max-width: 300px;
+}
+a {
+  text-decoration: none;
+  color: white;
+}
+.artist {
+  font-size: 14px;
+}
+
+.artistLink {
+color: grey;
+}
+.title:hover,
+.artistLink:hover {
+  text-decoration: underline;
+}
+.cover-image {
+  position: relative;
+}
+
+/* icons styling  */
+.icons {
+  position: absolute;
+  width: 75%;
+  justify-content: space-between;
+  align-items: center;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  display: none;
+  filter: invert(100%);
+}
+.dots-icon,
+.heart-icon {
+  width: 30px;
+  height: 30px;
+}
+.play-icon {
+  width: 50px;
+  height: 50px;
+  padding-left: 40px;
+  padding-right: 40px;
+  opacity: 0.7;
+}
+
+/* hover effects */
+.album:hover .image {
+  filter: brightness(0.3);
+}
+.album:hover .icons {
+  display: flex;
+}
+.play-icon:hover {
+  width: 60px;
+  height: 60px;
+  opacity: 1;
+}
+.artist a:not(:first-of-type, :last-of-type)::before {
+  color: grey;
+  content: ", ";
+}
+.artist a:nth-last-of-type(2)::after {
+  content: " & ";
+  color: grey;
+}
+
+/* footer styling */
+footer h4 {
+  font-family: "Ubuntu";
+  font-size: 1em;
+  color: grey;
+  border-top: 1px solid rgb(60, 60, 60);
+  text-align: center;
+  padding-top: 16px;
+}
+
+/* Media queries */
+@media (min-width: 768px) {
+  .container {
+    display: grid;
+    justify-items: center;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: repeat(2, 1fr);
```

Comment:
Is it necessary to have grid-template-rows when you already have grid-template-columns?